website_and_docs/content/documentation/overview/details.ja.md

```diff
@@ -1,104 +1,67 @@
 ---
-title: "A Deeper Look"
-linkTitle: "A Deeper Look"
+title: "Selenium 詳細"
+linkTitle: "詳細"
 weight: 2
 description: >
-  Seleniumはブラウザー自動化を可能にし、それを支えるツール群とライブラリー群のプロジェクトです。
+  Seleniumは、Web ブラウザーの自動化を可能にし、
+  サポートするさまざまなツールとライブラリーの包括的なプロジェクトです。
 aliases: [
-"/documentation/ja/introduction/the_selenium_project_and_tools/",
```

Comment:
These are old links that will get redirected to this page. It does not make sense to change them to `en` because it will either produce a random behavior, or the English link will end up on this page.

Another alternative is to remove the `aliases` section, since those links are very old. Up to you.

Reply:
@diemol Thank you for your comment. 
I removed the aliases section.

Reply:
Once I tried to remove the aliases section, but some checks below failed. So I add aliases section.